doc/classes/EditorVCSInterface.xml

```diff
@@ -4,112 +4,301 @@
-				Each [Dictionary] object has the line diff contents under the keys:
-				- [code]"content"[/code] to store a [String] containing the line contents
-				- [code]"status"[/code] to store a [String] which contains [code]"+"[/code] in case the content is a line addition but it stores a [code]"-"[/code] in case of deletion and an empty string in the case the line content is neither an addition nor a deletion.
-				- [code]"new_line_number"[/code] to store an integer containing the new line number of the line content.
-				- [code]"line_count"[/code] to store an integer containing the number of lines in the line content.
-				- [code]"old_line_number"[/code] to store an integer containing the old line number of the line content.
-				- [code]"offset"[/code] to store the offset of the line change since the first contextual line content.
+				Used to show line change gutters in script editor. [b]Currently not used by editor.[/b] Returns Array of Dictionary (see [method create_diff_hunk]).
 			</description>
 		</method>
-		<method name="get_modified_files_data">
-			<return type="Dictionary">
+		<method name="_get_modified_files_data" qualifiers="virtual">
+			<return type="Array">
+			</return>
+			<description>
+				This method is called when editor wants status of files. Equivalent to [code]git status[/code]. Returns Array of Dictionary (see [method create_status_file]).
+			</description>
+		</method>
+		<method name="_get_previous_commits" qualifiers="virtual">
+			<return type="Array">
 			</return>
 			<description>
-				Returns a [Dictionary] containing the path of the detected file change mapped to an integer signifying what kind of change the corresponding file has experienced.
-				The following integer values are being used to signify that the detected file is:
-				- [code]0[/code]: New to the VCS working directory
-				- [code]1[/code]: Modified
-				- [code]2[/code]: Renamed
-				- [code]3[/code]: Deleted
-				- [code]4[/code]: Typechanged
+				This method is called when editor needs list of previous commits. Returns Array of Dictionary (see [method create_commit]).
 			</description>
 		</method>
-		<method name="get_project_name">
+		<method name="_get_project_name" qualifiers="virtual">
 			<return type="String">
 			</return>
 			<description>
-				Returns the project name of the VCS working directory.
+				Virtual function to get project name or path as name.
 			</description>
 		</method>
-		<method name="get_vcs_name">
+		<method name="_get_vcs_name" qualifiers="virtual">
 			<return type="String">
 			</return>
 			<description>
-				Returns the name of the VCS if the VCS has been initialized, else return an empty string.
+				Virtual function to get name of VCS plugin.
```

Comment:
This function returns "Git" IIRC, and in that case it is actually returning the current VCS in use